okapi-core/src/main/raml/okapi.raml

```diff
@@ -46,6 +46,66 @@ traits:
         enum: ["true","false","only"]
         required: false
         default: "true"
+  installParameters:
+    queryParameters:
+      async:
+        description: Whether to install in the background
+        type: boolean
+        required: false
+        default: false
+      deploy:
+        description: Whether to deploy (or undeploy if disabling)
+        type: boolean
+        required: false
+        default: false
+      ignoreErrors:
+        description: |
+          Okapi 4.2.0 and later, it is possible to ignore errors during the
+          install operation. This is done by supplying parameter `ignoreErrors=true`.
+          In this case, Okapi will try to upgrade all
```

Comment:
Line breaks could be made more consistent, maybe at 100 chars.

Reply:
Yeah.. It's all over the place.